### ISSUE TITLE
Provide context for search results in Works Included and Objects Included (from 2031)

### ISSUE THREAD

**Issue by gigamorph** [project-lux/lux-frontend#85]
**Problem Description:**
The Works Included page lists the title of the search results but does not include any context that can help differentiate results from one another.  

**Expected Behavior/Solution:**
Change the search results on works pages to display like a [ result card search ](https://lux-front-tst.collections.yale.edu/view/concept/ac745260-c56a-45f3-ba9e-700ac0283434). This will allow users to gain context for search results in the 'Works Included' page.

The data for `This object includes the following works` on objects pages will show the first 5 and then a `Show All` button to view the remaining objects as there is no search link associated with this list.

**Requirements:**
- Show the first 5 results for initial display.
- Have a "show all [#] results" at the bottom of the initial result display. 
- When button is clicked user will be brought to an advanced search page that will list all results. 
- Objects will have a "Show all" button that will render the rest of the works in that page

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [x] Wireframe/Mockup - Mike
- [x] Feasibility/Team discussion - Sarah
- [ ] ~Committee discussions - Sarah~
- [x] Backend requirements - TBD
- [x] Frontend requirements- TBD
- [ ] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**

- Works example: https://lux-front-tst.collections.yale.edu/view/text/3da53ad9-478e-4395-b27a-71ad1520f5c5
- Object example: https://lux-front-tst.collections.yale.edu/view/object/b231e7dc-10bb-4a9f-ad2c-6fbaeddd87a3

- Used for wireframes/mocks - https://lux-front-tst.collections.yale.edu/view/text/3da53ad9-478e-4395-b27a-71ad1520f5c5

**Dependencies/Blocks:**

**Related Github Issues:**
- https://github.com/project-lux/lux-frontend/issues/86
- Old tix: https://git.yale.edu/lux-its/lux-web/issues/2031

**Related links:**
- https://www.bugherd.com/projects/284041/tasks/1962

**Wireframe/Mockup:**
What it looks like now: 
![image](https://github.com/project-lux/lux-frontend/assets/160149818/0ab0b2ba-afcb-4246-9412-d6ca86396f61)

What it should look like: 
![2031](https://github.com/project-lux/lux-frontend/assets/98550395/688dd313-9243-464d-a293-50d005469345)

**Comment by miketullo95** [comment]
@prowns updated wireframes

**Comment by kamerynB** [comment]
Can it be confirmed that this change will occur on both works pages and objects pages?

@kkdavis14 @prowns @miketullo95

**Comment by kkdavis14** [comment]
Yes, I think we'll want this on both.

**Comment by roamye** [comment]
Per UAT this is approved.

**Comment by roamye** [comment]
Closing as it looks good in prod. 

Note: @kamerynB 
I noticed that the show all button is included on "the object includes the following works"  but not in "the work includes the following objects" when there is only one result card.
 
Ex: 
https://lux.collections.yale.edu/view/text/88ee22af-0c56-4017-954e-e808661001c6
https://lux.collections.yale.edu/view/digital/6c8279d6-7091-4fef-be3d-63f62da522ca

We will need a hot fix for this in upcoming deployment. Will add a BH for this issue.
